src/Combinatorics/Matroids/matroids.jl

```diff
@@ -1019,3 +1019,48 @@ function automorphism_group(M::Matroid)
   resize!(I, nrows(I), length(M))
   return automorphism_group(I; action=:on_cols)
 end
+
+@doc raw"""
+
+	is_quotient(Q1::Matroid,Q2::Matroid)
+
+Let Q1 and Q2 be matroids on the groundset $[n]$, with rank $rank(Q2) = rank(Q1)+1$.  Check if $Q1$ is a \emph{matroid quotient} of $Q2$.
```

Comment:
```suggestion
Let Q1 and Q2 be matroids on the groundset $[n]$, with rank $rank(Q2) = rank(Q1)+1$.  Check if $Q1$ is a matroid quotient of $Q2$.
```
this does not support latex here